core/server/services/auth/api-key/content.js

```diff
@@ -34,7 +34,7 @@ const authenticateContentAPIKey = function authenticateContentAPIKey(req, res, n
         }));
     }
 
-    models.ApiKey.findOne({id: apiKeyId, type: 'content'}).then((apiKey) => {
+    models.ApiKey.findOne({id: apiKeyId}).then((apiKey) => {
```

Comment:
Why is this the fix, rather than removing the check in the `findOne` resolution handler?

Reply:
Better UX for API consumers as we can give specific error messages.